src/TestFramework/TestFramework/Assertions/CollectionAssert.cs

```diff
@@ -1280,6 +1487,7 @@ internal static bool IsSubsetOfHelper(ICollection subset, ICollection superset)
         return true;
     }
 
+#pragma warning disable CS8714
```

Comment:
Can't we have narrower scope pragma disable?

Reply:
no, i need it for both functions